### ISSUE TITLE
Cannot customize to save utc datetime for UpdateDateColumn

### ISSUE THREAD

**Issue by trantuat** [typeorm/typeorm#3637]
**Issue type:**

[x] question
[ ] bug report
[ ] feature request
[ ] documentation issue

**Database system/driver:**

[ ] `cordova`
[ ] `mongodb`
[ x] `mssql`
[ ] `mysql` / `mariadb`
[ ] `oracle`
[ ] `postgres`
[ ] `sqlite`
[ ] `sqljs`
[ ] `react-native`
[ ] `expo`

**TypeORM version:**

[ ] `latest`
[ ] `@next`
[ x] `0.2.11` 

**Steps to reproduce or a small repository showing the problem:**

Hi team,
On the version 0.2.11,
I have created a fileld call `updatedAt' with decorator `@UpdateDateColumn()` have type datetimeoffset.

export class Test {
    @PrimaryGeneratedColumn('uuid')
    id: string;

   @Column({
        type: 'nvarchar'
    })
    name: string;

    @UpdateDateColumn({
        type: 'datetimeoffset',
        default: () => 'getutcdate()'
    })
    updatedAt: Date;
}

My problem is, when I update record, the field updatedAt generate DateTime with database time. (The time saved is the time server host database, not an UTC DateTime). As I see on console log:
`UPDATE "Test" SET "name" = @1  "updatedAt" = CURRENT_TIMESTAMP OUTPUT INSERTED."updatedAt" WHERE "id" = @0`.
So how can I customize that the field updatedAt can save an UTC DateTime? 

Thanks!!

**Comment by vlapo** [comment]
I dont think so we have some "patch" for this issue now.  We are using different database type for `UpdateDateColumn` under postgres and everything is all right (mysql same I think). `mssql` do not convert standard sql function `CURRENT_TIMESTAMP` into right format.

This will requires some additional functionality for drivers. I dont know if this will be fixed in reasonable time. 

As workaround you have to define `updatedAt` as column and update it every time.

**Comment by L-Albano** [comment]
Has there been any updates towards this issue? I was recently attempting the same thing in version 0.2.20 and ended up with the same problem.

**Comment by Kannas24** [comment]
Can someone help me with [#6059](https://github.com/typeorm/typeorm/issues/6059)
